.github/workflows/build.yaml

```
@@ -15,7 +15,11 @@ jobs:
         uses: Randomblock1/theos-action@v1
 
       - name: Build package
-        run: make package
+        run: |
+          export PACKAGE_BUILDNAME=$(git rev-parse --short HEAD)-rootless
+          make package THEOS_PACKAGE_SCHEME=rootless FINALPACKAGE=1
```

I'll implement your suggestion so we can get this moved through
```suggestion
          make clean package THEOS_PACKAGE_SCHEME=rootless FINALPACKAGE=1
```